deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -449,7 +449,17 @@ master:
   #   requests:
   #     memory: "2Gi"
   #     cpu: "500m"
-
+  enableCustomizedConfig: false
+  customizedConfig: { }
```

Comment:
New configuration items need to be commented

Reply:
Add comments link to origin application.yaml path.

Reply:
Please refer to [helm doc](https://github.com/norwoodj/helm-docs) to write comments so that [helm doc](https://github.com/norwoodj/helm-docs) can automatically generate documentation

Reply:
Sure. Done.